### PR TITLE
Delete win7 and kvm info from v2v jobs

### PR DESCRIPTION
**Pull request by mxie91**
Signed-off-by: mxie91 <mxie@redhat.com>

# Format of PR title < sub-system: summary >
e.g
*virsh_migrate: Fix unsupported direct socket mode issue*

# Check lists by category
## If the PR is new cases
- [ ] Description of the cases
- [ ] Links of libvirt features, libvirt bugs or case IDs
- [ ] Test results

## If the PR is bug cases
- [ ] Bug descriptions or bug links
- [ ] Test results

## If the PR is a trivial fix
It it is the fix of typos, comments or documents, the description of PR could be skipped.

### CODE REVIEW

**Review by yantian0 — APPROVED** [review]

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 1]
```diff
@@ -65,7 +65,6 @@
         - raw_f:
             image_format_n = ""
         - qcow2_f:
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:67]: v2v only supports -of raw for -f rhv/rhv_upload right now. 
It's better to remove the whole '-qcow2_f' part.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 2]
```diff
@@ -65,7 +65,6 @@
         - raw_f:
             image_format_n = ""
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:66]: This 'image_format_n' is useless now, it's only for kvm convertion.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg (hunk 1 above)
Comment (xiaodwan) [reply v2v/tests/cfg/convert_vm_to_ovirt.cfg:67]: and this line 'qcow2_f..rhv_upload:output_format = "raw"'

**Comment by xiaodwan** [comment]
Please also add the reasons why we deleted those codes when we look back.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg (hunk 2 above)
Comment (mxie91) [reply v2v/tests/cfg/convert_vm_to_ovirt.cfg:66]: ok, I will delete it

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 3]
```diff
@@ -35,11 +35,12 @@
             rhv_upload_opts = "-oc ${ovirt_engine_url} -op ${rhv_upload_passwd_file} -oo rhv-cafile=${local_ca_file_path} -oo rhv-cluster=${cluster_name}"
             variants:
                 - preallocated:
-                    # bz1574734, bz1600547
-                    only dest_rhev.ISCSI,dest_rhev.FC
+                    # qcow2+preallocated will cause bz1939479
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:38]: I don't like those codes be added in tp-libvirt. Those cases have bugs should be put into skiplist in libvirt-ci.
So we don't have to change the tp-libvirt code again and again.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 4]
```diff
@@ -35,11 +35,12 @@
...
+                    only raw_f
                     rhv_upload_opts = "${rhv_upload_opts} -oa preallocated"
                 - nopreallocated:
-                    only dest_rhev.NFS
+                    # raw+sparse will cause bz1600547
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:42]: Same as above comment.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 5]
```diff
@@ -63,11 +64,7 @@
     variants:
         # This variable is only used for assembling vaule of main_vm
         - raw_f:
-            image_format_n = ""
         - qcow2_f:
-            only kvm
-            image_format_n = "-qcow2"
-            output_format = "qcow2"
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:70]: If you keep the qcow2_f part, this line 'output_format = "qcow2"' should be keep, too.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 6]
```diff
@@ -63,11 +64,7 @@
     variants:
         # This variable is only used for assembling vaule of main_vm
         - raw_f:
-            image_format_n = ""
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:66]: It's better to add 'output_format = "raw"' explicitly here.

**Comment by xiaodwan** [comment]
If you updated the code as I suggested, this line could also be deleted. 
'qcow2_f..rhv_upload:output_format = "raw"'

**Comment by mxie91** [comment]
@xiaodwan  Submit below two PR for the PR, please help to review them, thanks

https://gitlab.cee.redhat.com/libvirt-auto/libvirt-ci/-/merge_requests/1320/
https://gitlab.cee.redhat.com/libvirt-auto/ci-shared-datas/-/merge_requests/249

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**